### PR TITLE
User page

### PR DESCRIPTION
**Pull request by lubnaabd**
## Description
**front end to the user page**
user.css #25
  * File contain style user Page.

**back end to the home page**
view_user.hbs: #26
 - Page contain table view all data in user
  * user name
  * address
  * phone

 - Page contain table view all Borrowed user
  * user name
  * address
  * phone
  * startDate
  * endDate
### Type of change

New features #67
Description : front-end view_user.

New features #70
Description : front-end view_Borrowed User.

New features #11
Description : 
* create new router for the viewUser.
* create new router for the Borrowed User.

### CODE REVIEW

**Review by ali-7 — APPROVED** [review]

**Review by asmaa-ezz — APPROVED** [review]

File: src/views/view_user.hbs [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<section class="main__header">
+  <ul class="main__header--nav">
+    <li onclick="viewUser(event,'user')" class="main__header--nav-tablink">
+      <a data-toggle="tab" aria-expanded="false" href="/admin/users">
+        الأعضاء
+      </a>
+    </li>
+    <li onclick="viewUser(event,'borower')" class="main__header--nav-tablink">
+      <a data-toggle="tab" aria-expanded="false" href="/admin/borrowedUsers">المستعريين</a>
```
Comment (RamyAlshurafa) [inline src/views/view_user.hbs]: المستعريين => المستعيرين
:smiling_imp:

**Review by RamyAlshurafa — CHANGES_REQUESTED** [review]
Pull the master branch and add your routes into the controller

File: src/controllers/index.js [hunk 2]
```diff
@@ -1,8 +1,12 @@
 const express = require('express');
 const adminHomePage = require('./admin_homePage.js');
+const user = require('./user.js');
```
Comment (RamyAlshurafa) [inline src/controllers/index.js:3]: you can use destructuring here

File: src/controllers/index.js [hunk 3]
```diff
@@ -1,8 +1,12 @@
...
 
 const router = express.Router();
 
 router.get('/admin/', adminHomePage.get);
+router.get('/admin/users', user.getViewUser);
+router.get('/admin/borrowedUsers', user.getBorrowedUser);
+
```
Comment (RamyAlshurafa) [inline src/controllers/index.js:10]: remove this line

File: src/controllers/user.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+      viewUser: 'active',
+      user: 'active',
+      layout: 'admin',
+      title: 'اﻷعضاء',
+      style: 'user',
+      js: 'user',
+    });
+};
+
+
```
Comment (RamyAlshurafa) [inline src/controllers/user.js]: remove this line

**Review by RamyAlshurafa — CHANGES_REQUESTED** [review]

**Review by RamyAlshurafa — COMMENTED** [review]
next time: use different commit message for each commit

**Review by RamyAlshurafa — APPROVED** [review]

**Pull request merged**